aep/general/0002/aep.md

```diff
@@ -0,0 +1,64 @@
+# AEP Numbering and Organization
+
+This document describes the numbering system and general organization of AEPs.
+
+AEP numbers:
+
+- Are unique. No two AEPs share the same number.
+- Indicate the order in which they are authored.
+
+To help with organization and discovery, AEPs also have a category and a slug
+to easily discovery and refer to the AEP.
+
+## AEP Numbers
+
+### Core AEPs
+
+To uniquely identify an AEP, each one is assigned a permanent number once it
+has been merged into the aep.dev repository. The AEP number assigned **must**
+be the next available number. AEPs **must** be numbered between 1 and 9999.
+
+### Organization-specific AEPs
+
+Organizations **may** extend the AEPs with guidance that is relevant to them.
+When doing so, they **must** use AEP numbers 10000 or greater.
+
+## AEP Categories
+
+To help with discovery of an AEP, each one is assigned a category: a string the
+describes a theme or grouping the proposal belongs in. The category of a
```

Comment:
"proposal" as in the "P" in "AEP"?  Should we just say AEP here?

Reply:
I thought it might be better to have a separate term for a single AEP, and thought "proposal" might work. In case we decide to change names again.

But I could just use AEP everywhere, if you feel strongly about it.

Reply:
I just updated everything to say "AEP" explicitly.